src/components/app.jsx

```diff
@@ -25,24 +25,20 @@ const App = React.createClass({
           <div className="container" id="content" style={{marginBottom: 100}}>
             <section className="jumbotron" style={{marginTop: "1em"}}>
               <div className="row">
-                <div className="col-lg-3">
-                  <p>
-                    <img
-                      alt="Logo OpenFisca"
-                      className="img-responsive"
-                      src={url.resolve(config.websiteUrl, "/hotlinks/logo-openfisca.svg")}
-                    />
-                  </p>
+                <div className="col-md-3">
+                  <img
+                    alt="Logo OpenFisca"
```

Comment:
You have to think of the `alt` attribute as a text replacement that will be read to a screenreader, for example. Nobody cares that it's a logo.
As [decorated text](https://www.w3.org/QA/Tips/altAttribute), the `alt` attribute should have only a value of “OpenFisca”, not “Logo OpenFisca”. 
However, in the current case, since you repeat “OpenFisca” in the sentence right after it (which might not be so useful, btw), the image is decorative only and you should use an [empty `alt`](https://www.w3.org/TR/WCAG20-TECHS/H67.html).